AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/EBeanHolder.java

```diff
@@ -58,6 +62,16 @@ private void setConstructor() {
 			constructorBody.invoke("super").arg(constructorContextParam);
 		}
 		constructorBody.assign(getContextField(), constructorContextParam);
+
+		overloadedConstructor = generatedClass.constructor(PRIVATE);
+		JVar overloadedConstructorContextParam = overloadedConstructor.param(getClasses().CONTEXT, "context");
+		JVar overloadedConstructorRootFragmentParam = overloadedConstructor.param(getClasses().OBJECT, "rootFragment");
+		JBlock overloadedConstructorBody = overloadedConstructor.body();
+		if (superConstructor.getParameters().size() == 1) {
+			overloadedConstructorBody.invoke("super").arg(constructorContextParam);
+		}
```

Comment:
Travis fails because of this i think. You have to write:
```java
overloadedConstructorBody.add(JExpr.invokeSuper().arg(constructorContextParam));
```